app/assets/src/components/views/pipeline_viz.scss

```diff
@@ -4,38 +4,62 @@
 .pipelineViz {
   display: flex;
   justify-content: center;
+  align-items: center;
 }
 
 .stage {
   display: flex;
   justify-content: center;
   align-items: center;
-  margin: 24px;
+  margin: 24px 0;
 }
 
 .stageButton {
   background-color: $light-grey;
   border-radius: 8px;
   padding: 12px;
-  width: 120px;
+  width: 250px;
   text-align: center;
   font-weight: $font-weight-bold;
   cursor: pointer;
 }
 
+.stageArrow {
```

Comment:
Would this not be better as a React component that lets you pass in the width of the arrow as a prop? Or will stage arrows always be static?

Reply:
For now stage arrows are static, and are only used in the context of this specific part of the graph so not sure if it's worth creating a whole separate component